extensions/smallrye-graphql/deployment/src/main/java/io/quarkus/smallrye/graphql/deployment/SmallRyeGraphQLProcessor.java

```diff
@@ -483,7 +484,8 @@ void getGraphqlUiFinalDestination(
                                         LINE_FORMAT)
                                 .getBytes(StandardCharsets.UTF_8);
                         content = WebJarUtil
-                                .updateUrl(new String(content, StandardCharsets.UTF_8), graphQLPath, UI_LINE_TO_UPDATE,
```

Comment:
This changes `graphQLPath` for `graphQLUiPath`, is it on purpose?

Reply:
Yes, was a bug in the prev PR. This is used in "home" (image) click